### ISSUE TITLE
New copilot chat icons are of different width doesn't look good

### ISSUE THREAD

**Issue by abhijit-chikane** [microsoft/vscode#229693]
Earlier
![image](https://github.com/user-attachments/assets/c9b65d1d-c261-4bc0-bd14-09fa60019c7e)

Newer
![image](https://github.com/user-attachments/assets/c128e2de-8f2d-41db-8b3f-e8ca956589b8)

The new icons has different width see the @ and send icons width is larger than other two icons which doesn't look good.
Not sure why we have introduced these two skinny icons even though they are taking the same 22 * 22 space

**Comment by daviddossett** [comment]
@roblourens I think this is mostly due to the toolbars being split and causing the uneven gap

**Comment by roblourens** [comment]
I think the @ is visually wider than the paperclip and mic too

**Comment by daviddossett** [comment]
Yeah, but that's basically impossible to avoid given what the icons are. Maybe tilting the attach icon again would help here? I'll give it a try.

**Comment by abhijit-chikane** [comment]
Also the earlieer mic icon is having the similar width like @
